src/test/java/org/example/ByteValidatorTest.java

```diff
@@ -1,23 +1,40 @@
 package org.example;
+
 import static org.junit.jupiter.api.Assertions.*;
 import org.junit.jupiter.api.Test;
+
 public class ByteValidatorTest {
 
     @Test
-    public void testValidByteArray() {
-        String validCharacters = "Hello";
-        ByteValidator validator = new ByteValidator(validCharacters);
+    public void testValidByteArrayISO() {
+        // ISO-8859-9 karakter seti ile bir ByteValidator oluştur
+        ByteValidator validator = new ByteValidator("ISO-8859-9");
 
-        byte[] validArray = { 'H', 'e', 'l', 'l', 'o' };
+        byte[] validArray = {(byte) 0xE7};
```

Comment:
Thisis not easy to understand. Use `string.getBytes()` method for a more readable test.

For example: `"öşğüçı".getBytes()`